### PR TITLE
Regenerate with latest gir

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gdk-sys/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [build-dependencies]
 pkg-config = "0.3.7"
```
Comment (sdroege) [inline gdk-sys/Cargo.toml:2]: This will cause a CI failure: https://github.com/gtk-rs/gir/pull/937/files#r446032219

@EPashkin What do you suggest?

**Comment by sdroege** [comment]
Related gstreamer-rs-sys MR is here: https://gitlab.freedesktop.org/gstreamer/gstreamer-rs-sys/-/merge_requests/61

Nice removal of repetitive code :)

**Comment by sdroege** [comment]
Also CC @gdesmott

File: gdk-sys/Cargo.toml (hunk 1 above)
Comment (sdroege) [reply gdk-sys/Cargo.toml:2]: Apart from that everything works

Comment (EPashkin) [reply gdk-sys/Cargo.toml:2]: https://github.com/gtk-rs/gir/pull/941

Comment (sdroege) [reply gdk-sys/Cargo.toml:2]: Ah, @EPashkin fixed already :)

**Comment by sdroege** [comment]
@GuillaumeGomez All green

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**